CHANGELOG.md

```diff
@@ -1,19 +1,30 @@
 # Changelog
 
+## 0.50.0
+
+### Changes
+
+- Use the `githubPullRequests.notifications` setting to highlight Pull Requests which have unread notifications
```

Comment:
Would be great to have a screen shot for this! And to mention the setting that can be used to turn this on/off.